gulpfile.js

```diff
@@ -105,6 +105,22 @@ gulp.task('start-containers-dev', function(){
     }); 
 });
 
+gulp.task('start-containers-prod', function(){
+    
+    var adminP;
+    if (!fs.existsSync('./api/clients/Admin-browserify-bundle.js'))
```

Comment:
Now it hits me, whit the `build` task run before, all the file checking is useless. Same for `start-container-dev`